database-portfolio/src/main/webapp/script.js

```diff
@@ -13,43 +13,36 @@
 // limitations under the License.
 //
 
+/** Loads and displays comments according to a limit */
 function loadCommentSection(commentLimit) {
-  
     // clear previous elements
     var currentDOM = document.getElementsByClassName('indiv-comments');
-    console.log(currentDOM);
     if (currentDOM != null) {
         console.log(currentDOM);
         Array.from(currentDOM).forEach(function (element) {
-            console.log(element);
             element.remove();
         })
     }
 
-    commentURL = '/list-comments?limit=' + commentLimit;
+    commentURL = `/list-comments?limit=${commentLimit}`
     fetch(commentURL)
-    .then(response => response.json())
-    .then((comments) => {
-
-    /* Build the list of comments.
-    */
-    const commentLe = document.getElementById('post-content');
-
-    comments.forEach((comment) => {
-      commentLe.appendChild(createCommentElement(comment));
-    })
-
-  });
+      .then(response => response.json())
+      .then((comments) => {
+      // Build the list of comments..
+      const commentLe = document.getElementById('post-content');
+      comments.forEach((comment) => {
```

Comment:
I recommend using a for(let comment of comments) {} loop instead. ForEach is actually not allowed in google3 code.